esphome/components/mcp4461/output/mcp4461_output.cpp

```diff
@@ -0,0 +1,150 @@
+#include "mcp4461_output.h"
+#include <cmath>
+
+#include "esphome/core/helpers.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace mcp4461 {
+
+static const char *const TAG = "mcp4461.output";
+
+void Mcp4461Wiper::write_state(float state) {
+  if (this->parent_->is_failed()) {
+    ESP_LOGW(TAG, "Parent MCP4461 component has failed! Aborting");
```

Comment:
```suggestion
    ESP_LOGW(TAG, "%s", LOG_STR_ARG(LOG_PARENT_FAILED_STR));
```

static const LogString *const LOG_PARENT_FAILED_STR = LOG_STR("Parent MCP4461 component has failed! Aborting");

https://github.com/esphome/esphome/pull/8131#pullrequestreview-2574087032

Take a look at this, you can save quite a bit of flash by doing it this way.

Similar is done here: https://github.com/esphome/esphome/blob/9e3359cdf23685b6b6b9c4df4c824c0037847ed7/esphome/components/ens210/ens210.cpp#L133